File: src/main/java/no/novari/flyt/authorization/user/AccessControlProperties.java
package no.novari.flyt.authorization.user;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.context.annotation.Configuration;

import java.util.Map;

@Getter
@Setter
@Configuration
@ConfigurationProperties(prefix = "novari.flyt.authorization.access-control")
public class AccessControlProperties {
    private Map<String, String> permittedAppRoles;
}

File: src/main/kotlin/no/novari/flyt/authorization/user/AccessControlProperties.kt
package no.novari.flyt.authorization.user

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "novari.flyt.authorization.access-control")
data class AccessControlProperties(
    var permittedAppRoles: Map<String, String> = emptyMap(),
)
